rpcs3/Emu/SysCalls/Modules/cellMusic.cpp

```diff
@@ -99,9 +110,48 @@ s32 cellMusicSelectContents()
 	throw EXCEPTION("");
 }
 
-s32 cellMusicInitialize2()
+s32 cellMusicInitialize2(s32 mode, s32 spuPriority, vm::ptr<CellMusic2Callback> func, vm::ptr<void> userData)
 {
-	throw EXCEPTION("");
+	cellMusic.Todo("cellMusicInitialize2(mode=%d, spuPriority=%d, func=*0x%x, userData=*0x%x)", mode, spuPriority, func, userData);
+
```

Comment:
@Nekotekina The way the docs are worded, would make me think that the function returns basically right away and errors and completion of the function are notified through the callback. I would say it would be still safer to call the callback, even if you return an error, as a game could be stupid enough to expect callback to be called for notification of processing to continue, even if the function itself returned an error.